core_api/src/app.py

```diff
@@ -154,6 +137,7 @@ async def create_upload_file(file: UploadFile, ingest: bool = True) -> File:
     if ingest:
         await ingest_file(file_record.uuid)
 
+    # TODO: return something more sensible to the user (including errors)
```

Comment:
For a 200, what should we return to the user - just 'your file has been uploaded successfully?'

Reply:
i dont see why not, even nothing would be fine?